### PR TITLE
build_manual function

### PR DESCRIPTION
**Pull request by twolodzko**
Added build_manual function as discussed in  #1238.

### CODE REVIEW

File: R/build-manual.R [hunk 1]
```diff
@@ -0,0 +1,19 @@
+
```
Comment (hadley) [inline R/build-manual.R]: Can you please eliminate the extra newlines here and after the docs?

File: R/build-manual.R [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+#'   \code{\link{as.package}} for more information.
+#' @param path path in which to produce package manual.
+#'   If \code{NULL}, defaults to the parent directory of the package.
+#'
+#' @seealso \code{\link{RdUtils}}
+#' @export
+
+build_manual <- function(pkg = ".", path = NULL) {
+  pkg <- as.package(pkg)
+  if (is.null(path)) {
```
Comment (hadley) [inline R/build-manual.R]: This can be written more succinctly as:

```R
path <- path %||% dirname(pkg$path)
```

File: R/build-manual.R [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+    path <- dirname(pkg$path)
+  }
+  name <- paste0(pkg$package, "_", pkg$version, ".pdf", collapse = " ")
+  system(paste0("R CMD Rd2pdf --force --output=", path, "/", name, " ", shQuote(pkg$path), collapse = " "))
```
Comment (hadley) [inline R/build-manual.R]: Can you please use `callr::rmd()` here?

Comment (twolodzko) [inline R/build-manual.R]: I made the two other edits you suggested, but I don't understand the syntax for `callr::rcmd` and I can't make it work.

Comment (hadley) [inline R/build-manual.R]: It should look something like 

```R
callr::rcmd("Rd2pdf", args = c(
  "--force", 
  paste0("--output=", path, "/", name), 
  pkg$path
))
```

Comment (twolodzko) [inline R/build-manual.R]: @hadley thanks, done.

**Comment by hadley** [comment]
Thanks for all your work on this!

**Comment by lfmcmillan** [comment]
I'm really glad there is a build_manual() function as a separate function, because when I run check(manual=TRUE) and build(manual=TRUE) from the command line in Windows 10, they don't actually create the manual, and running build_manual() seems to be the only way to successfully create the PDF manual

**Pull request merged**